documentation/assemblies/security/assembly-security.adoc

```diff
@@ -14,9 +16,6 @@ The Cluster Operator automatically sets up and renews TLS certificates to enable
 It also sets up other TLS certificates if you want to enable encryption or TLS authentication between Kafka brokers and clients.
 Certificates provided by users are not renewed.
```

Comment:
This seems a bit out of place since we do not really mention that users can provide their own certificates. In general, it might be good mentioning it here in short with some link for more details. Similarly, I would probably keep the paragraph below about the listener certificates. These are fairly common questions.

Reply:
Listener certs are back. I've rewritten:

> If you don't want to use the certificates generated by the Cluster Operator, you can xref:installing-your-own-ca-certificates-str[install your own cluster and client certificates].
> 
> You can also xref:kafka-listener-certificates-str[provide _Kafka listener certificates_] for TLS listeners or external listeners that have TLS encryption enabled. Use Kafka listener certificates to incorporate the security infrastructure you already have in place.

Reply:
> If you don't want to use the certificates generated by the Cluster Operator, you can xref:installing-your-own-ca-certificates-str[install your own cluster and client certificates].

This is not completely right. It is just the CA which you provide. The CO will still generate the certificates. So it should be something like `If you don't want to use the CAs generated by the Cluster Operator, you can xref:installing-your-own-ca-certificates-str[install your own cluster and client CAs].`.